pkg/services/images.go

```
@@ -130,6 +130,20 @@ func (s *ImageService) CreateImage(image *models.Image, account string, orgID st
 	if image.Version == 0 {
 		image.Version = 1
 	}
+	packages := image.Packages
+	arch := image.Commit.Arch
+	dist := image.Distribution
+	ib := imagebuilder.Client{}
+	// we now need to loop this request for each package
+	for _, p := range packages {
+		res, err := ib.SearchPackage(p.Name, arch, dist)
+		if err != nil {
+			return err
```

we should not return this error, as the validation errors are returned to user, as the client is logging it's errors we do not need  to log here , we can return a global error as "Unknown validation error occurred"